spec/abstractops.html

```diff
@@ -1442,4 +1442,190 @@ <h1>ParseTemporalYearMonthString ( _isoString_ )</h1>
         }.
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-temporal-topositiveintegerorinfinity" aoid="ToPositiveIntegerOrInfinity">
+    <h1>ToPositiveIntegerOrInfinity ( _argument_ )</h1>
+    <p>
+      The abstract operation ToPositiveIntegerOrInfinity takes argument _argument_. It converts _argument_ to +&infin; or an integer _i_ for which <emu-eqn>0 &lt; i</emu-eqn>. It performs the following steps:
```

Comment:
```suggestion
      The abstract operation ToPositiveIntegerOrInfinity takes argument _argument_. It converts _argument_ to +∞ or an integer _i_ for which <emu-eqn>0 &lt; i</emu-eqn>. It performs the following steps:
```

Reply:
I noticed https://github.com/tc39/ecma262/blob/master/spec.html uses &infin; and prefer it myself because I can type it rather than paste it. Alright to keep?

Reply:
I also prefer character entities in general, but every so often @Ms2ger goes through and replaces any that have crept in (except for `&lt;` and `&gt;`), so I've assumed there's an editorial requirement or preference not to use them. (I type ∞ with <kbd>Caps Lock</kbd>, <kbd>8</kbd>, <kbd>8</kbd>, if it helps)